refactron/cli.py

```
@@ -528,20 +686,57 @@ def autofix(
+        console.print(f"[dim]📋 Using template: {template}[/dim]")
+        if template != "base":
+            console.print(
+                f"[dim]💡 Template includes framework-specific settings for {template}[/dim]"
+            )
+        console.print("\n[dim]Edit this file to customize Refactron behavior.[/dim]")
+        console.print(
+            "[dim]💡 Use --profile or --environment options to switch between dev/staging/prod[/dim]"
+        )
+    except ValueError as e:
+        console.print(f"[red]❌ Error: {e}[/red]")
+        raise SystemExit(1)
```

Ruff (0.14.13)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)